### ISSUE TITLE
Use VS Code Web Custom Data package for HTML definitions

### ISSUE THREAD

**Issue by janechu** [microsoft/fast#3772]
# Abstract

Currently the Component Definitions use the same structure as the [VS Code custom data format](https://github.com/microsoft/vscode-custom-data). In order to expand on this so include all standard HTML elements for the sites Component Explorer and Creator the definitions must be available

## Requirements

To keep up-to-date with the latest web standards a dependency should be taken on [VS Code Web Custom Data](https://www.npmjs.com/package/vscode-web-custom-data). This dependency should include definition files for all HTML elements.

- Include the above npm package in the `sites/site-utilities`
- Remove the definitions from the `src/definitions/native` folder and replace them with exports from [VS Code Web Custom Data](https://www.npmjs.com/package/vscode-web-custom-data)
- Test the Creator and Component Explorer sites to ensure this is working as intended.

Note: In the event that this creates so many attributes that the `Form` becomes unweildy to use, instead create custom definitions for native elements for now, open a new task for that and link this one. We may need to complete this task after #2575

**Further edits**
## Additional notes

The `"slots"` are an added part to the schema and were taken as inspiration from https://github.com/webcomponents/custom-elements-json/issues/3, there is a file located in the `@microsoft/fast-tooling`, `data-utilities/html-elements.ts` which includes a string array of the elements that do not contain content. As part of this task, use this array to selectively add the default slots to all other HTML elements.
